index.html

```diff
@@ -0,0 +1,170 @@
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon3.png" alt="icon" />
+            <h3 class="programheadtext">Dialoque</h3>
+            <p class="programtext">
+              Have that game that is still stuck in your mind from months ago? This is the time to talk about with playes who can provide you with all sorts of perspective
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon4.png" alt="icon" />
+            <h3 class="programheadtext">PLANS</h3>
+            <p class="programtext">
+              The next next agenda point is the discussing of strategies, are there any new opening moves or strategies, or is it still the old timers who have the advantage.
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon5.png" alt="icon" />
+            <h3 class="programheadtext">MINGLE</h3>
+            <p class="programtext">
+              Personally, this is always the main advantage of these kind of meetups for me, i learn so much from all the different eople i meet during theese events.
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div class="programlinkwrapper">
+        <a class="programlink" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <section class="speakerssect">
+      <div class="announce">
+        <div class="announcetext">Featured speakers</div>
+        <hr class="announcehr" />
+      </div>
+      <div class="speakers"></div>
+    </section>
+    <section class="partnerssec">
+        <div class="announce">
+            <div class="announcetext">Partner</div>
+            <hr class="announcehr" />
+          </div>
+     <div class="partnerwrapper">
+         <ul class="partners">
+             <li><img src="assets/images/navar.png" alt=""></li>
+             <li><img src="assets/images/duamkako.png" alt=""></li>
+             <li><img src="assets/images/Google.png" alt=""></li>
+             <li><img src="assets/images/airbnb.png" alt=""></li>
+             <li><img src="assets/images/mozilla.png" alt=""></li>
+         </ul>
+     </div>
+
+
+    </section>
+    <section class="footer">
+        <footer>
+            <div class="footerleft">
+                <a href="index.html"><img class="logo botlog" src="assets/images/creativecommons.png" alt="" /></a>
+            </div>
+            <div class="footerright">
+                <p>2015 Creative Commons Korea,Some Rights Reserved</p>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Reprehenderit, atque itaque, aliquid corrupti ducimus pariatur ipsa sunt alias repudiandae laudantium accusantium dolorum voluptas nam culpa qui saepe magni voluptatum placeat.</p>
```

Comment:
- [ ] The footer has a margin too big and also the color is different from the design.

- **Original footer design**
![roig](https://user-images.githubusercontent.com/61562626/137174746-316c7120-eaf4-45fa-a3c2-e79c18e1d39d.jpeg)


- **Your design**

![mqmlq](https://user-images.githubusercontent.com/61562626/137174791-472c04bc-a734-49dd-8cad-8808cb145242.jpeg)